backend/src/apiserver/server/util.go

```diff
@@ -268,3 +273,36 @@ func CheckPipelineVersionReference(resourceManager *resource.ResourceManager, re
 
 	return &pipelineVersionId, nil
 }
+
+func GetUserIdentity(ctx context.Context) (string, error) {
```

Comment:
please add tests for new code

Reply:
Added tests for GetNamespaceFromResourceReferences and Authorize

Reply:
Constructing a fake grpc context for GetUserIdentity test is a little complex and the tests codes might be error-prone, Do you think it is worth it? If so, I can do that.

Reply:
Done.